File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L2/05/fungsi.py
import sys

def terbalik(angka):
    while angka != 0:
        sisa = angka % 10
        print(sisa, end='')
        angka = angka // 10
    print()

if __name__ == "__main__":
    print("Enter an integer: ", end='')
    input_int = int(sys.stdin.readline().strip())
    terbalik(input_int)

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L5/06/fungsi.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function terbalik(angka: number): void {
  while (angka > 0) {
    const sisa = angka % 10;
    console.log(sisa);
    angka = Math.floor(angka / 10);
  }
  console.log();
}

rl.question('Enter an integer: ', (input) => {
  const inputNumber = parseInt(input);
  terbalik(inputNumber);
  rl.close();
});

